sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -214,16 +214,15 @@ TextAnalyticsAsyncClient textAnalyticsClient = new TextAnalyticsClientBuilder()
 ```
 
 ### Detect language
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L75-L82 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L76-L82 -->
 ```java
 String inputText = "Bonjour tout le monde";
 
-for (DetectedLanguage detectedLanguage : textAnalyticsClient.detectLanguage(inputText).getDetectedLanguages()) {
-    System.out.printf("Detected languages name: %s, ISO 6391 Name: %s, Score: %s.%n",
-        detectedLanguage.getName(),
-        detectedLanguage.getIso6391Name(),
-        detectedLanguage.getScore());
-}
+DetectedLanguage detectedLanguage = textAnalyticsClient.detectLanguage(inputText).getPrimaryLanguage();
```

Comment:
This will change once PR https://github.com/Azure/azure-sdk-for-java/pull/7939/files#diff-120a7e45ac3baf2e0719201e59beb2e4R220 goes in.